Toolset/palettes/message box/behaviors/revmessageboxbehavior.livecodescript

```diff
@@ -667,8 +667,8 @@ private command __UpdateIntelligenceDisplay
    else
       local tControl
       put the name of it into tControl
-      set the label of button "Open Stacks" of group "Stacks" of me to tIntelligence & ": " & tControl
-      enable button "Open Stacks" of group "Stacks" of me
+      set the label of button "Open Stacks" of group "Stacks" of card "Single Line" of me to tIntelligence & ": " & tControl
+      enable button "Open Stacks" of group "Stacks" of card "Single Line" of me
```

Comment:
I think this should be either Single Line or Multiple Lines depending on which is the current card

Reply:
Perhaps we should have a behavior above the single and multiple line card behaviors that deals with this instead, but that's beyond the scope of this PR ;-)

Reply:
The group is shared but it is only on some cards. We just need to make sure we reference one of the cards its on here and it will work fine. The issue is only if the user is on some other card and the message comes in.